### DISCUSSION TITLE
$AssimpFbx$_Translation and _PreRotation nodes

### DISCUSSION THREAD

**Discussion by dtecta** [assimp/assimp#4966] [Q&A]
I'm using AssImporter 5.1.0 and loading character rigs from https://www.mixamo.com. Previously these FBXs would load fine. However, lately AssImporter is adding additional  bones. I'm using the same AssImporter version. Why are these extra bones added and how can I avoid them from being added? Thanks for your help.

**Comment by dtecta** [comment]
The property 

importer.SetPropertyBool(AI_CONFIG_IMPORT_FBX_PRESERVE_PIVOTS, false); 

prevents generation of pivot nodes.

**Reply by fernandotonon** [reply]
On my side it is also creating these bones, even with the importer.SetPropertyBool(AI_CONFIG_IMPORT_FBX_PRESERVE_PIVOTS, false);

<img width="1055" alt="image" src="https://user-images.githubusercontent.com/996529/221066022-8bd6a1a2-b537-4032-958f-9351df6b2bcb.png">
